### DISCUSSION TITLE
README Thank-You Section

### DISCUSSION THREAD

**Discussion by ROBdk97** [Dymerz/StarCitizen-Localization#20] [Ideas]
Expressing gratitude to those who played a vital role in the initial translation for the project is not only a gesture of appreciation for their time and effort but also a recognition of the potential financial investment they committed.

**Comment by Dymerz** [comment]
That a great idea, I could easily add the bot from https://allcontributors.org 
I will do that!

**Reply by Dymerz** [reply]
I've just setup the @all-contributors bot and contribution section 🎉
